src/test/java/com/work/management/service/employee/impl/EmployeeServiceImplTest.java

```diff
@@ -22,6 +29,10 @@
 public final class EmployeeServiceImplTest {
 
   private static final Employee EMPLOYEE = getEmployee();
+  private static final List<Employee> EMPLOYEE_LIST = getEmployeeList();
+  private static final Map<AcceptedFields, String> ACCEPTED_FIELDS = getAcceptedFields();
+  private static final List<Integer> EMPLOYEE_ID_LIST = getEmployeeIdList();
```

Comment:
variable type should be as specific as possible, make them ImmutableList and ImmutableMap

Reply:
If a variable is not getting used across test case like Bulk_employee_dto move it to that test case only.
Only make static final variables if that variable is getting used across test cases.
Similarly for others.